test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -590,8 +590,8 @@ private void DoWorkspaceOpen(string filePath)
         private static void SaveJsonTempWithFolderStructure(string filePath, JObject jo)
         {
             // Get all folder structure following "\\test"
-            int subStringIndex = filePath.IndexOf("\\test") + 5;
-            var expectedStructure = filePath.Substring(subStringIndex);
+            var testDirectory = SerializationTests.TestDirectory;
+            var expectedStructure = filePath.Split(new string[] { testDirectory }, StringSplitOptions.None).Last();
```

Comment:
Wouldn't it be easier to just remove the first `testDirectory.Length()` characters from filePath?

Reply:
@gregmarr do you also need .ds and .data files or just .dyn and .dyf?

Reply:
I need .ds and .data

Reply:
So one thing to look out for is a .dyn and a .dyf in the same directory with the same base name: Example.dyn and Example.dyf.  The .data and .ds files for the .dyf would conflict with the ones from the .dyn if/when they're generated.

Reply:
@gregmarr I am assuming modifying the `.dyf` file name will break various tests.  Would appending something such as `_dyf` to the `.data` or `.ds` file names for all custom nodes also cause issues?

Reply:
I suggested doing something like that, but someone else suggested moving them to another directory instead.  I'm not sure what tests would have to be changed for that.